cumulus-client/src/main/java/mil/army/usace/hec/cumulus/client/controllers/CumulusFileDownloaderUtil.java

```diff
@@ -0,0 +1,60 @@
+package mil.army.usace.hec.cumulus.client.controllers;
+
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.channels.Channels;
+import java.nio.channels.FileChannel;
+import java.nio.file.Path;
+import mil.army.usace.hec.cumulus.client.model.Download;
+import mil.army.usace.hec.cwms.http.client.ApiConnectionInfo;
+import mil.army.usace.hec.cwms.http.client.HttpRequestBuilderImpl;
+import mil.army.usace.hec.cwms.http.client.HttpRequestResponse;
+import mil.army.usace.hec.cwms.http.client.request.HttpRequestExecutor;
+
+public final class CumulusFileDownloaderUtil {
```

Comment:
This class is public, but none of the methods are public.

Reply:
shouldn't be public. Updated.